.travis.yml

```diff
@@ -1,12 +1,31 @@
 language: php
 
 php:
-  - 5.3.3
-  - 5.3
   - 5.4
   - 5.5
   - 5.6
+  - hhvm-nightly
 
-install: composer install -n
+before_install:
+    - sh -c "sudo mkdir vendor"
+    - sh -c "sudo mount -t tmpfs -o size=512M tmpfs vendor"
+
+install:
+    - composer install --prefer-source --no-interaction
+
+before_script:
+    - composer self-update
+    - sh -c "echo 'extension = redis.so' >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini"
+    - sh -c "sudo mkdir /tmp/Bamboo"
+    - sh -c "sudo mount -t tmpfs -o size=512M tmpfs /tmp/Bamboo/"
+
+services:
+    - redis-server
+
+script:
+    - ./vendor/behat/behat/bin/behat
+    - bin/behat
```

Comment:
Why both?

The `bin/behat` file is copied into `bin/` folder via composer, so `bin/behat` should be enough, right?